### ISSUE TITLE
Inds no top screen with AirShou

### ISSUE THREAD

**Issue by EdwardElric64** [WilliamLCobb/iNDS#214]
So I'm trying to do a lets play series with iNDS on my iPad and I'm trying to use AirShou to record it but whenever I start it up the top screen is missing, is it because of the AirPlay? Can AirShou not be used with iNDS?

**Comment by noah978** [comment]
iNDS projects the top screen to airplay for use with apple TV's. Which might prevent you from being able to use iNDS with AirShou.

**Comment by EdwardElric64** [comment]
Is there anything I can donor non jailbreak? As I used build store to get all of this

**Comment by xMilesManx** [comment]
Hey I feel like I need to point out a couple of things. 

First off the issues tab isn't a "help" forum. It is meant to help find and debug issues in the code. 
Second of all, this project hasn't been updated in over 8 months, so don't expect perfect compatibility with the latest iOS version. I would also imagine that this project is about to be depreciated since he has announced he is working on iNDS 2.0 

Last but not least, you kinda got scammed with the ibuildstore. You can simply compile this from the source and install it for a week on your device. If you don't know how to do that, a simple google search will suffice.

**Comment by EdwardElric64** [comment]
I got build store 6 months before Mach portal was a thing, you really think I would have payed $10 if I knew such a thing existed? Second I didn't know he was working on iNDS 2.0 until a few hours ago, and third there was no need to be rude about it, you could have just simply said the issue tab is not for help

**Comment by xMilesManx** [comment]
Alright man take it down a few notches here. I didn't say anything about a jailbreak. You can simply compile the application from the source code and install it to your device. You don't need a jailbreak to do that. The obvious downside is that you need to keep resigning it every week(if you don't know how to do this I would highly suggest do some research as it is very helpful to be able to do this for some other projects). With buildstore I think it's for a year? But you wouldn't get any of the latest changes in the code build as they come out. I wasn't being rude I was simply pointing out a few things that you may have missed and that this really shouldn't be used as a help forum.

**Comment by EdwardElric64** [comment]
I don't know how to compile from source, and I know how to resign

**Comment by xMilesManx** [comment]
It's a great skill to have. I would highly suggest learning to compile it. You can use it and never need a jailbreak!

**Comment by EdwardElric64** [comment]
Would you mind linking me a video on how to do it?

**Comment by noah978** [comment]
I would suggest operationidroid's video on how to use Xcode with open-sourced emus. The first thing you need to know about it is that it's not possible without a mac. https://www.youtube.com/watch?v=hfno95Jraew. Just put some additional research into the subject and you should be able to learn how to use Xcode. Good luck!

**Comment by EdwardElric64** [comment]
But I don't own a mac... possible with windows?

**Comment by WilliamLCobb** [comment]
I can send you a build that fixes this while I work on something more permanent. There's just one line to comment out. How are you downloading and installing iNDS?

**Comment by EdwardElric64** [comment]
I'm using buildstore because I don't know how to compile with source and I'm being told I can't do that without a mac

**Comment by noah978** [comment]
The only way Xcode is possible with windows, is using a Virtual Machine to emulate masOS. However, the process of obtaining and setting up a virtual machine typically requires a great deal of expertise. If you want to try, there are a few guides, but I couldn't find one specific to your situation in a quick google search.

**Comment by EdwardElric64** [comment]
Dang... I thought there was an Xcode for windows though?

**Comment by noah978** [comment]
There is. Only if your willing to pay $99.99 per year

**Comment by EdwardElric64** [comment]
If my laptop which is windows, which has 12 GB of ram, gen 7 i7 processor, and a 4 GB gaming Graphics card, would I be able to run a virtual machine?

**Comment by EdwardElric64** [comment]
Graphics card is a GeForce

**Comment by noah978** [comment]
Yes, I've been able to run a virtual machine on an i5 intel processor using intel graphics.

First, https://techsviewer.com/enable-intel-virtualization-technology-vt-x-bios/.

Second, https://techsviewer.com/install-macos-sierra-vmware-windows/. The macOS might be missing its serial number.

**Comment by EdwardElric64** [comment]
I have not been able to get to installing and side loading yet, but when I went to activate virtualization technology I realized I already had it active from when I had blue stacks XD

**Comment by EdwardElric64** [comment]
I was finally able to get virtual machine, but when I go to run the Mac OS after saying virtual machine it's just a black screen, i seen people have a problem where their virtual machine just reboots, and in the video by techs review people are saying re enter the code from step 5, do I just do that?

**Comment by noah978** [comment]
I would send an email or comment to them, as I don't specialize in virtual machines.

**Comment by EdwardElric64** [comment]
Finally, I got it working, although I ended up using VMware in the end, but Xcode is now downloading
